Assets/Live2D/Cubism/Rendering/CubismRenderer.cs

```diff
@@ -515,15 +515,19 @@ public void UpdateVisibility()
         /// <summary>
         /// Updates render order.
         /// </summary>
-        public void UpdateRenderOrder()
+        /// <returns>is sorted.</returns>
+        public bool UpdateRenderOrder()
         {
+            var res = false;
             if (LastSwap.NewRenderOrder)
             {
                 ApplySorting();
+                res = true;
             }
 
 
             ResetRenderOrderFlag();
+            return res;
```

Comment:
We will put this on hold for now, as it is related to `Assets/Live2D/Cubism/Rendering/CubismRenderController.cs`.